CODEOWNERS

```diff
@@ -42,6 +42,7 @@
 /soc/arm/nuvoton/                         @ssekar15
 /soc/arm/qemu_cortex_a53/                 @carlocaione
 /soc/arm/st_stm32/                        @erwango
+/soc/arm/silabs_exx32/efr32mg21/          @l-alfred
```

Comment:
Please move this up a line so it's not stuck between two st_stm32 entries

Reply:
This is done now, thanks for your input @MaureenHelm